pkgs/top-level/all-packages.nix

```diff
@@ -15739,12 +15739,8 @@ with pkgs;
 
   pipewire = callPackage ../development/libraries/pipewire {
     # ffmpeg depends on SDL2 which depends on pipewire by default.
-    # Break the cycle by disabling pipewire support in our ffmpeg.
-    ffmpeg = ffmpeg.override {
-      SDL2 = SDL2.override {
-        pipewireSupport = false;
-      };
-    };
+    # Break the cycle by depending on ffmpeg-headless.
+    ffmpeg = ffmpeg-headless;
```

Comment:
What does pipewire use ffmpeg for? Just want to make sure we're not giving it an ffmpeg that's missing critical functionality.

Reply:
It's only used in https://github.com/PipeWire/pipewire/blob/master/spa/plugins/ffmpeg/ffmpeg.c and friends, and just for encoding/decoding, so only `libavcodec`. The `-headless` changes shouldn't affect this (except _maybe_ the change to `libva-minimal`, but I seriously doubt that would hurt).

Reply:
Might be good to add that as a comment here.